open-hackathon-server/src/hackathon/views/resources.py

```diff
@@ -680,8 +681,31 @@ def put(self):
     def delete(self):
         return docker_host_manager.delete_host_server(self.context().id)
 
+
 class AdminHackathonCanOnLineResource(HackathonResource):
     @admin_privilege_required
     def get(self):
         return hackathon_manager.check_hackathon_online(g.hackathon)
 
+
+class AdminHackathonNoticeResource(HackathonResource):
+    @hackathon_name_required
+    def get(self):
+        return hackathon_manager.get_hackathon_notice(g.hackathon, self.context().type)
```

Comment:
json file for parameter validation for both `AdminHackathonNoticeResource` and `HackathonNoticeListResource`